src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.BlockValidationTransactionsExecutor.cs

```diff
@@ -33,6 +33,8 @@ public BlockValidationTransactionsExecutor(ITransactionProcessorAdapter transact
 
             public TxReceipt[] ProcessTransactions(Block block, ProcessingOptions processingOptions, BlockReceiptsTracer receiptsTracer, IReleaseSpec spec)
             {
+                Evm.Metrics.ResetBlockStats();
```

Comment:
Isn't this the only place you want to do that? Other are not in main block processing loop.

Reply:
Kept, removed others